IncludeToolbox/Commands/FormatIncludes.cs

```diff
@@ -79,7 +79,7 @@ protected override void MenuItemCallback(object sender, EventArgs e)
                     line.UpdateTextWithIncludeContent();
 
                 // Sorting. Ignores non-include lines.
-                IncludeFormatter.IncludeFormatter.SortIncludes(lines, settings.SortByType, settings.RegexIncludeDelimiter, settings.PrecedenceRegexes, document.Name);
```

Comment:
Got to be too many settings required so I just passed in the whole settings object.